src/web/app/components/question-types/question-edit-answer-form/mcq-scale-question-edit-answer-form.component.html

```diff
@@ -0,0 +1,21 @@
+<tr>
+  <td>
+    <div class="radio" *ngFor="let num of questionDetails.mcqChoices; let i = index;">
+      <label class="margin-right-15px">
+        <input type="radio" class="margin-right-15px" name="check_{{ id }}" [checked]="isMcqOptionSelected[i]"
+        (click)="updateSelectedMcqOption(i)">
+         <strong>{{ questionDetails.mcqChoices[i] }}</strong>
+      </label>
+    </div>
+    <div class="radio" *ngIf="questionDetails.otherEnabled">
+      <label class="margin-right-15px">
+        <input type="radio" class="margin-right-15px" name="check_{{ id }}" [checked]="responseDetails.isOther"
+        (click)="updateIsOtherOption()">
+        <strong>Random Text Other</strong>
```

Comment:
This should not be what is shown to the user...?

Reply:
Only Other is meant to be shown.